.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,6 @@ quickly and easily:
  - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
```

Comment:
hey dipti - it looks like in this commit you did a merge instead of a rebase - we can't accept PRs that have old commits by other people in them, so we can't use this PR as-is.

I'm not sure if it's easier for you to:
1. use git cherry-pick to pull the commits from this branch into a new branch (and potentially start a new PR?)
2. use git reset to rewind back to before all these changes and then re-apply your changes 
(or something else I haven't thought of :)

regardless, we can't have these changes in this PR